packages/typespec-ts/test/integration/generated/unions/src/outputModels.ts

```diff
@@ -45,4 +45,5 @@ export interface MixedTypesCasesOutput {
   boolean: CatOutput | "a" | number | boolean;
 }
 
+/** Alias for StringExtensibleNamedUnionOutput */
```

Comment:
add description for union.

Reply:
just curios, why it is introduced by this pr, not named union one?

Reply:
oh i see the changes in rlc-common